library/std/src/os/xous/services/dns.rs

```diff
@@ -0,0 +1,30 @@
+use crate::os::xous::ffi::Connection;
+use crate::os::xous::services::connect;
+use core::sync::atomic::{AtomicU32, Ordering};
+
+#[repr(usize)]
+pub(crate) enum DnsLendMut {
+    RawLookup, /* 6 */
+}
+
+impl Into<usize> for DnsLendMut {
+    fn into(self) -> usize {
+        match self {
+            DnsLendMut::RawLookup => 6,
+        }
+    }
+}
```

Comment:
Maybe do it like this?

```suggestion
#[repr(usize)]
pub(crate) enum DnsLendMut {
    RawLookup = 6,
}

impl Into<usize> for DnsLendMut {
    fn into(self) -> usize {
        self as usize
    }
}
```

Reply:
Good idea. I'd like to be able to do it for more complex enums, but better to do it where possible.